modules/services/karabiner-dk/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.karabiner-dk;
+
+  parentAppDir = "/Applications/Nix Apps";
```

Comment:
We should add a way to reference this instead of redefining otherwise we'll have to change the directory path in multiple places